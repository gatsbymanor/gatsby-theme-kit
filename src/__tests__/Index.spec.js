import React from 'react';
import renderer from 'react-test-renderer';
import Index from '../pages/index';

describe('<Index />', () => {
  const component = renderer.create(<Index />).toJSON();

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});