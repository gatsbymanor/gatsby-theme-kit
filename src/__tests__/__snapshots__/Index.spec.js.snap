// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index /> should render correctly 1`] = `
<div
  id="wrapper"
>
  <Header />
  <Nav />
  <Main>
    <Intro />
    <First />
    <Second />
    <GetStarted />
  </Main>
  <Footer />
</div>
`;
